app/javascript/src/components/Dashboard/Notes/NoteItem/NoteItem.jsx

```diff
@@ -45,9 +45,12 @@ const NoteItem = ({
         />
         <div className="flex items-center">
           <Clock color="#1e1e20" size={22} />
-          <Tooltip content={formatDate(createdAt)} position="bottom-end">
+          <Tooltip
+            content={formatDateWithDayAndTime(createdAt)}
+            position="bottom-end"
+          >
             <Typography className="mx-2" style="body3">
-              {lastUpdated(createdAt)}
+              {formatDateRelativeToNow(createdAt)}
```

Comment:
Where is the `formatDateRelativeToNow` defined? Had you done a self-review from UI before merging this PR?